File: capivaraprojects/greeneyes/models/Disease.py
from capivaraprojects.greeneyes.models.Plant import Plant
class Disease:
    def __init__(self, id=0, plant=Plant(), scientificName="", commonName="", images=list()):
        self.id = id
        self.plant = plant
        self.scientificName = scientificName
        self.commonName = commonName
        self.images = images

File: capivaraprojects/greeneyes/models/Disease.js
const Plant = require('./Plant.js');

class Disease {
	constructor (id=0, plant=Plant, scientificName="", commonName="", images=list()) {
		this.id = id;
		this.plant = plant;
		this.scientificName = scientificName;
		this.commonName = commonName;
		this.images = images;
	}
}